pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/filesystem/ListFiles.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools.admin.command.filesystem;
+
+import java.io.IOException;
+import java.net.URI;
+import org.apache.pinot.spi.filesystem.PinotFSFactory;
+import org.apache.pinot.spi.utils.StringUtil;
+import picocli.CommandLine;
+
+
+@CommandLine.Command(name = "ls", aliases = {"list"}, description = "List files")
+public class ListFiles extends BaseFileOperation {
+  @CommandLine.Option(names = {"-r", "--recursive"},
+      description = "Recursively list subdirectories")
+  private boolean _recursive;
+
+  @CommandLine.Parameters(arity = "1..*", description = "File paths to list")
+  private String[] _filePaths;
```

Comment:
do we really want ppl to list multiple paths?

Reply:
seems ls supports multiple parameters